Game.js

```diff
@@ -43,40 +43,40 @@ export default class Game extends React.Component {
   //   }
   // }
 
-  render() {	
+  render() {
     // Determine the size (in pixels) of a single Box.
     var {width, height} = Dimensions.get('window')
-    width -= 100;
```

Comment:
Why did they remove 100 and add 50?

Reply:
Stop cyber-bullying others online. Not very nice Justin Shao. Even though you don't understand does not give you the right to bully others.

Reply:
cuz 100 is a dumb number

Reply:
comment

Reply:
what is a smart number?

Reply:
ur mom dumb number